web/src/Web.App/Views/SchoolComparators/Index.cshtml

```diff
@@ -71,7 +71,32 @@
                     </tr>
                     </thead>
                     <tbody class="govuk-table__body">
-
+                    
+                    @if (Model.PupilSchools.Any(x => x.URN == Model.Urn))
```

Comment:
This outer `if` seems redundant. Can we do the conditional block on `chosenSchool` instead?